### DISCUSSION TITLE
Change location of stair layout

### DISCUSSION THREAD

**Discussion by Jack-Whattam** [anametologin/krohnkite#106] [General]
When tiling windows in stair layout it defaults to the right side of a given screen with a single window at the left side (i.e., a two-column layout with stair layout in the right-hand column). I find this general layout really useful but for my screen set up it would work better to have the stair layout on left-hand side of the screen.

Whilst I can manually move windows around this is quite tedious if I have to do so after every reboot. Is there a way to change the default behaviour or to move multiple windows to different locations?

**Comment by anametologin** [comment]
Have you tried change direction in `Stair layout` settings? If so, could you post a screenshot of what you want .

**Reply by Jack-Whattam** [reply]
Sorry for the delayed response!

I did some more experimenting and it turns out I was mistaken as to which layout was which and it is actually the tile layout that I am working with. So is there a way to have the stacked windows of the tile layout appear on the left of the screen, with a single column window on the right (the opposite of the arrangement in the screenshot below)?

![Krohnkite_Screen](https://github.com/user-attachments/assets/2f7349b1-7d04-46cb-8964-bad73423abf5)

**Comment by anametologin** [comment]
there is `initial rotation angle` under tile layout configuration.
